resources/views/widgets/settings/graylog.blade.php

```diff
@@ -29,6 +29,22 @@
         <input type="number" min="1" class="form-control" name="limit" id="limit-{{ $id }}" placeholder="@lang('Page Size')" value="{{ $limit }}">
     </div>
 
+    <div class="form-group">
+        <label for="loglevel-{{ $id }}" class="control-label">@lang('Log Level')</label>
+        <select name="loglevel" id="loglevel-{{ $id }}" class="form-control">
+            <option value="" disabled @if($loglevel == null) selected @endif>@lang('Log Level')</option>
+            <option value="0" @if($loglevel === 0) selected @endif>@lang('(0) Emergency')</option>
```

Comment:
These should use the translation strings `syslog.severity.0`

Reply:
Changed this. Did not realize (also never tried) that I can use these translation Strings there.

Reply:
You could even use a foreach so you don't have to repeat every option ;)

Reply:
I can't get this to work with the translation strings.
```
@foreach(range(0, 7) as $level)
                <option value="{{ $level }}" @if($loglevel == $level && $loglevel != null) selected @endif>({{ $level }}) @lang('syslog.severity.{{ $level }}')</option>
            @endforeach
```
Results in a select field with `(0) syslog.severity.` and so on as options,

```
@foreach(range(0, 7) as $level)
                <option value="{{ $level }}" @if($loglevel == $level && $loglevel != null) selected @endif>({{ $level }}) @lang('syslog.severity.'{{ $level }})</option>
            @endforeach
```
results in `Problem with backend`